### PR TITLE
docs: update SDK code for GreptimeCloud console

### PR DESCRIPTION
**Pull request by nicecui**
## What's Changed in this PR

* update SDK code
* add Java example code about Influxdb line protocol integration

## Checklist

- [x] Please ensure that the content in `summary.yml` matches the current document structure when you changed the document structure.
- [ ] This change requires follow-up update in localized docs.

### CODE REVIEW

File: docs/v0.6/en/greptimecloud/integrations/influxdb.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ Please refer to [InfluxDB client](https://docs.greptime.com/user-guide/clients/i
 - Username: `<username>`
 - Password: `<password>`
 
+For example, the following example writes a data point to the `system_metrics` table:
+
 ```sh
 curl -i 'https://<host>/v1/influxdb/write?db=<dbname>&u=<username>&p=<password>' \
 --data-binary 'system_metrics,host=host1,idc=idc_a cpu_util=11.8,memory_util=10.3,disk_util=10.3 1667446797450000000'
```
Comment (sunng87) [inline docs/v0.6/en/greptimecloud/integrations/influxdb.md:18]: I think we can add an example of using an influxdb client for this, using any programming language, just to demo the configuration.

Comment (nicecui) [reply docs/v0.6/en/greptimecloud/integrations/influxdb.md:18]: researching and practicing, it will take some time...

**Comment by nicecui** [comment]
@sunng87 @tisonkun PTAL

File: docs/v0.6/en/greptimecloud/integrations/influxdb.md [hunk 2]
```diff
@@ -11,9 +11,10 @@ Please refer to [InfluxDB client](https://docs.greptime.com/user-guide/clients/i
 - Username: `<username>`
 - Password: `<password>`
 
-For example, the following example writes a data point to the `system_metrics` table:
+The following Java example code demonstrates how to configure the InfluxDB client when connecting to GreptimeCloud:
```
Comment (sunng87) [inline docs/v0.6/en/greptimecloud/integrations/influxdb.md:14]: Add a link to this java influxdb client's github page

**Review by sunng87 — APPROVED** [review]

**Review by Aganivi — APPROVED** [review]

**Pull request merged**